### PR TITLE
Share screen design review

### LINKED ISSUE #17569: Design Review: Share Screen
- [ ] 1. Toast has wrong theme - comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=6117:66732&mode=design#570499944)
- [ ] 2. Either the button is too close to the top or the toast is in the wrong place - comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=5960:8254&mode=design#567186677). - Was not able to reproduce on iPhone 11 Pro Simulator & Pixel 7a.
- [ ] 3. Fix spacing - comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=6117:66732&mode=design#570500481)
- [ ] 4. Wrong user image in the middle of the share screen qrcode - comment [here](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=5960:8254&mode=design#567186888)
  - Reproducible when the user doesn't have a profile picture

### PR DESCRIPTION
**Pull request by alwx**
Fixes https://github.com/status-im/status-mobile/issues/17569
Some of the issues were fixed before already, this PR fixes the rest.

#### Platforms

- Android
- iOS

status: ready

### CODE REVIEW

File: src/status_im2/contexts/shell/share/events.cljs [hunk 1]
```diff
@@ -10,8 +10,8 @@
   (rf/merge cofx
             {:copy-to-clipboard text-to-copy}
             (toasts/upsert
-             {:icon           :correct
-              :id             :successful-copy-toast-message
-              :icon-color     colors/success-50
-              :override-theme :dark
-              :text           post-copy-message})))
+             {:icon       :correct
```
Comment (smohamedjavid) [inline src/status_im2/contexts/shell/share/events.cljs]:
```suggestion
             {:icon  :i/correct
```

**Review by smohamedjavid — APPROVED** [review]

**Review by siddarthkay — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
hi @alwx thank you for PR. No issues from my side
Hey @Francesca-G could you take a look this PR, please?

**Review by Francesca-G — APPROVED** [review]
Looks good ✨

Only issue I spotted: the toast height doesn't match the design and the icon isn't aligned with text. I've encountered this issue recently in other PRs, please make sure it gets addressed, thank you :)

Screenshot 2023-12-15 alle 16 37 41

**Comment by alwx** [comment]
@Francesca-G I think changing toasts doesn't really fit the definition of what needs to be done in this particular issue. This is a separate issue for it: https://github.com/status-im/status-mobile/issues/17540

**Comment by Francesca-G** [comment]
> @Francesca-G I think changing toasts doesn't really fit the definition of what needs to be done in this particular issue. This is a separate issue for it: #17540

Yes, I just wanted to make sure it wasn't missed. Thanks for linking the separate issue for it 🙏

**Pull request merged**